Code/Source/VehicleDynamics/ChassisConfiguration.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "AxleConfiguration.h"
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzFramework/Physics/Material/PhysicsMaterial.h>
+
+namespace VehicleDynamics
+{
+    //! Drive and steering configuration for a vehicle
```

Comment:
Is this the right description for 'Chassis' ?

Reply:
I changed the class name and naming overall to vehicle configuration